trimmer_decorator.rb

```diff
@@ -0,0 +1,10 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  MAXIMUM_NAME_LENGTH = 10
+
+  def correct_name
+    # name = @nameable.correct_name
+    name.length > MAXIMUM_NAME_LENGTH ? name[0...MAXIMUM_NAME_LENGTH] : name
```

Comment:
- [ ] Kindly use the `@nameable.correct_name` as inherited from the Decorator class. This returns errors as you are using `name` which is defined nowhere.

  ```
  @nameable.correct_name.length > MAXIMUM_NAME_LENGTH ? @nameable.correct_name[0...MAXIMUM_NAME_LENGTH] : @nameable.correct_name
  ```